tonic/src/status.rs

```diff
@@ -748,14 +748,16 @@ impl From<std::io::Error> for Status {
 
 impl fmt::Display for Status {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        write!(
-            f,
-            "status: {:?}, message: {:?}, details: {:?}, metadata: {:?}",
-            self.code(),
-            self.message(),
-            self.details(),
-            self.metadata(),
-        )
+        write!(f, "status: '{}'", self.code())?;
+
+        if !self.message().is_empty() {
+            write!(f, ", self: {:?}", self.message())?;
```

Comment:
@LucioFranco should this be
```suggestion
            write!(f, ", message: {:?}", self.message())?;
```
?

Reply:
🤦

Reply:
Fix in:
* https://github.com/hyperium/tonic/pull/2414